hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3GuardOutOfBandOperations.java

```diff
@@ -549,10 +566,16 @@ public void deleteAfterTombstoneExpiryOobCreate() throws Exception {
       guardedFs.delete(filePath, true);
 
       // FILE MUST NOT EXIST IN RAW
-      intercept(FileNotFoundException.class, filePath.toString(),
-          "This file should throw FNFE when reading through "
-              + "the raw fs, and the guarded fs deleted the file.",
-          () -> rawFS.getFileStatus(filePath));
+      // If authoritative, the file status can be retrieved
```

Comment:
now I'm confused. shouldn't deletion through a guarded FS remove it from raw too? Or is it because the tombstone is in the auth dir that we don't bother to pass down the delete?
If that's the case -which I think it is- can you make that clear in the comments. It won't be obvious to anyone when we come back to this code in 2020

Reply:
I need to debug, something odd is happening. I don't know why don't we delete the file from raw though guarded.

Reply:
We get a FNFE when we call `guardedFs.getFileStatus(filePath)` if `authoritative`, but not when we call `rawFS.getFileStatus(filePath)`. Which means.. deleting is eventual in aws s3? That's not how it supposed to be.